newsfragments/43368.significant.rst

```diff
@@ -0,0 +1,5 @@
+Deprecated trigger rule removed
+
+**Breaking Change**
+
+The trigger rule prior to TriggerRule.ALWAYS has been deprecated and now removed.
```

Comment:
```suggestion
The trigger rule prior to TriggerRule.DUMMY has been deprecated and now removed.
```

Reply:
@eladkal I had to use the phrasing "prior to TriggerRule.ALWAYS" because the pre-commit prevents using `dummy`

Reply:
Then we need to fix the pre commit. You can add exclusion for this specific file